site/content/blog/2021-07-01-whats-new-in-svelte-july-2021.md

```diff
@@ -0,0 +1,56 @@
+---
+title: What's new in Svelte: July 2021
+description: Keeping cool with fixes, TypeScript tooling and tonnes of new features
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+As the northern hemisphere heats up, Svelte has stayed cool with lots of performance and bug fixes, better TypeScript support, and lots of new components & tools from around the ecosystem. Let's take a peek 👀
+
+## New in SvelteKit
+- `adapter-node` now precompresses assets using gzip & brotli ([#1693](https://github.com/sveltejs/kit/pull/1693))
+- Support for TypeScript transpilation has been added to the svelte package tooling ([#1633](https://github.com/sveltejs/kit/pull/1633))
+
```

Comment:
- Improved caching defaults in `adapter-node` ([#1416](https://github.com/sveltejs/kit/pull/1416))
- Allow configuring Rollup output options ([#1572](https://github.com/sveltejs/kit/pull/1572))
- Fixed usage of SSL with HMR ([#1517](https://github.com/sveltejs/kit/pull/1517))